gitlab/v4/objects/services.py

```diff
@@ -267,7 +267,7 @@ def get(
         return cast(ProjectService, super().get(id=id, lazy=lazy, **kwargs))
 
     @cli.register_custom_action("ProjectServiceManager")
-    def available(self, **kwargs: Any) -> List[str]:
+    def available(self) -> List[str]:
```

Comment:
Same as above, we should pass it on for `sudo` etc IMO.

Reply:
Pass it to what?

It just calls:
```
return list(self._service_attrs.keys())
```

Reply:
Oh I'm dumb, I just saw `list()` and assumed it was the same as above :grin: thanks.